apps/backend/src/modules/users/user-details/user-details.model.ts

```diff
@@ -0,0 +1,35 @@
+import { Model } from "objection";
+
+import {
+	AbstractModel,
+	DatabaseTableName,
+} from "~/libs/modules/database/database.js";
+
+import { UserModel } from "../user.model.js";
+
+class UserDetailsModel extends AbstractModel {
+	public static relationMappings = () => {
+		return {
+			user: {
+				join: {
+					from: `${DatabaseTableName.USER_DETAILS}.userId`,
+					to: `${DatabaseTableName.USERS}.id`,
+				},
+				modelClass: UserModel,
+				relation: Model.BelongsToOneRelation,
+			},
+		};
+	};
+
+	public static tableName: string = DatabaseTableName.USER_DETAILS;
```

Comment:
```suggestion
	public static override get tableName(): string {}
```
Let's have the same syntax everywhere